bubblewrap.c

```diff
@@ -2041,6 +2041,65 @@ read_overflowids (void)
     die ("Can't parse /proc/sys/kernel/overflowgid");
 }
 
+static void
+write_id_for_namespace (int         out_fd,
+                        int         pid_fd,
+                        const char *ns,
+                        bool        in_json)
+{
+  cleanup_free char *output = NULL;
+  cleanup_free char *dir = NULL;
+  const char *indent;
+  struct stat st;
+  uintmax_t nsid;
+  int r;
+
+  dir = xasprintf ("ns/%s", ns);
+  r = fstatat (pid_fd, dir, &st, 0);
+
+  if (r != 0)
+    die_with_error ("stat /proc/<pid>/%s failed", dir);
+
+  nsid = (uintmax_t) st.st_ino;
+  indent = in_json ? " " : "\n    ";
+  output = xasprintf (",%s\"%s-namespace\": %ju", indent, ns, nsid);
```

Comment:
Its a bit unclear why we're indenting the two differently. Both are json anyway. Seems like pointless complexity. @cgwalters do you remember if there was a reason for the current discrepancy?